templates/case-study/elementary-ai-machine-vision.html

```diff
@@ -0,0 +1,246 @@
+      </p>
+    {%- endif -%}
+  {%- endcall -%}
+
+  {%- call(slot) text(title="Challenge") -%}
+    {%- if slot == 'content' -%}
+      <p>
+        In 2017, Elementary was founded to transform manufacturing quality control with AI-powered computer vision. Before AI-powered systems were a reality, traditional machine vision systems were the go-to technique for quality control. While these machine vision systems were useful, they were also brittle. They relied on rigid, rules-based setups and often failed to function in real-world conditions – for example, when there were changes in light levels or variations in the products being inspected. These minor variations in conditions often resulted in the system becoming misconfigured, requiring a specialist in machine vision to reconfigure the system manually. As Peter Lipay, Vice President of Engineering at Elementary, explained: “We've often found we’re not the first machine vision system that our customers are deploying. They have one sitting there, unplugged, because it stopped working at some point. They just stopped using it, because it was too much of a pain to keep it in sync with what they were doing.”
+      </p>
+      <p>
+        Elementary’s approach has significantly improved machine vision systems. Its AI models can be trained on real-world data, and are flexible enough to adapt to changes in the operating environment or working conditions. For one of Elementary's Fortune 500 customers, this reduced defect rates from 24% to 1%. Additionally, IoT connectivity enables cloud analytics that provide greater oversight and control for manufacturers. However, as Elementary scaled, it outgrew its infrastructure. Running traditional server-based architecture on edge devices meant Elementary carried the heavy burden of OS updates, provisioning, and field support. Updates, configuration, and provisioning all required physical access to devices, increasing operational expenditure and creating data security concerns.
+      </p>
+    {%- endif -%}
+  {%- endcall -%}
+
+  {%- call(slot) quote_wrapper(
+    quote_text="In the past, Elementary was responsible for provisioning each of the devices for our customers – we had to intervene with any type of issues on behalf of the customer. We needed to move to a path where customers could actively take on that role and be able to perform device recovery themselves if need be.",
+    citation_source_name_text="Nathaniel Black",
+    citation_source_title_text="Senior Director of DevOps and Security",
+    citation_source_organisation_text="Elementary",
+    is_shallow=True,
+    hide_image_small=True
+    )
+    -%}
+  {%- endcall -%}
+
+  <section class="p-section">
+    <div class="row">
```

Comment:
```suggestion
    <div class="grid-row">
```

We should use the [new grid system](https://vanillaframework.io/docs/patterns/grid) now as the [legacy grid](https://vanillaframework.io/docs/patterns/grid-legacy) is being deprecated.